index.html

```diff
@@ -6195,6 +6195,35 @@ <h2>A project for first-time contributions</h2>
     </div>
     <!-- ________ Prashant Singh's card END ________  -->
 
+     <!-- ________ Felipe Siqueira's card START ________  -->
+     <div class="card">
+
+        <p class="name">Felipe Siqueira</p>
+        <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://www.instagram.com/felipe_sqra/" target="_blank">@felipe_sqra</a>
+        </p>
+        <p class="about">
+            I'm a Final Year Computer Science Undergraduate Student from Brazil. I'm passionate about learning and trying different things.
+        </p>
+        <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+                <li>
+                    <a href="#" target="_blank" title="https://www.sololearn.com/">Sololearn</a>
+                </li>
+                <li>
+                    <a href="#" target="_blank" title="https://www.udacity.com/">Udacity</a>
+                </li>
+                <li>
+                    <a href="#" target="_blank" title="https://www.udemy.com/">Udemy</a>
```

Comment:
You need to put the URL in the `href` attribute `href="https://www.udacity.com"`. Right now you have a `#` where the URL is supposed to be. The `title` attribute on the other hand only provides a little description that appears as a tooltip when the user hovers over the link. You will need to correct all three links for me to be able to merge your PR please.